docs/guides/self-signed-ca-certs.md

```diff
@@ -0,0 +1,50 @@
+---  
+title: Fetch Packages from Registry with Self-Signed CA Certificate  
+toc: true  
+weight: 270  
+indent: true  
+---  
+
+# Overview of Crossplane for Registry with Self-Signed CA Certificate  
+
+Crossplane uses the go-containerregistry remote pkg as the client to fetch
+package manifests and images from OCI registries. It uses the 
+http.DefaultTransport, meaning that it loads the default system certificates.
+This precludes fetching packages from private registries that may use certs
+from a self-signed CA.
+
+The default transport can be overridden with one that passes additional certs
+to the TLSConfig. This allows for self-signed certificate authorities to be
```

Comment:
nitpick: technically the TLS cert is what is self-signed, and it is coming from a custom (untrusted) CA

Reply:
change 1certificate authorities` to `certificate`